doc/changelog.rst

```diff
@@ -17,6 +17,22 @@ PyMongo 4.16 brings a number of changes including:
 - Removed support for Eventlet.
   Eventlet is actively being sunset by its maintainers and has compatibility issues with PyMongo's dnspython dependency.
 
+Changes in Version 4.15.4 (2025/10/21)
```

Comment:
is this also bundling the release alongside?

Reply:
I assumed it would have XXXX/XX/XX, since it hasn't been released yet. This isn't a blocking issue since it'll get updated when its going out.